CHANGELOG.en-US.md

```diff
@@ -0,0 +1,119 @@
+---
+order: 3
+title: Change Log
+toc: false
+timeline: true
+---
+
+If you want to read change logs before `2.0.0`, please visit [GitHub](https://github.com/ant-design/ant-design/releases?after=2.0.0).
+
+---
+
+很高兴的通知各位，经过四个月时间的紧密开发，`antd@2.0.0` 终于发布了。这个版本我们重构了底层代码，持续完善现有组件功能和优化细节，其中很多都来自社区的贡献，无法一一感谢，欢迎各位持续关注和鞭策。在升级过程中遇到任何问题，请及时反馈给我们。
+
+### 2.x Major changes
+
+* Refactor components with TypeScript, and provide **`.d.ts` files which are supported officially**. Thank all the developers in [#1846](https://github.com/ant-design/ant-design/issues/1846) and @infeng.
```

Comment:
* Refactor components with TypeScript, and provide **`.d.ts` files which are officially supported**. Thanks to all the developers that contributed to [#1846](https://github.com/ant-design/ant-design/issues/1846) and @infeng.